src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs

```diff
@@ -175,10 +179,10 @@ private Rune(uint scalarValue, bool unused)
         /// </summary>
         public int Value => (int)_value;
 
-        private static Rune ChangeCaseCultureAware(Rune rune, TextInfo textInfo, bool toUpper)
+        private static Rune ChangeCaseCultureAware(Rune rune, CultureInfo culture, bool toUpper)
```

Comment:
This should remain `TextInfo` in the coreclr case. Otherwise the caller isn't able to take advantage of a bunch of JIT / AOT optimizations that we introduced in the runtime.